python/acquire/__init__.py

```diff
@@ -8,13 +10,32 @@
 
 import logging
 
-FORMAT = (
+if TYPE_CHECKING:
+    import napari  # type: ignore
+
+
+FORMAT: str = (
     "%(levelname)s %(name)s %(asctime)-15s %(filename)s:%(lineno)d %(message)s"
 )
 logging.basicConfig(format=FORMAT)
 logging.getLogger().setLevel(logging.INFO)
 
 
+g_runtime: Optional[Runtime] = None
+"""The global acquire runtime."""
+
+
+def _get_runtime() -> Runtime:
```

Comment:
I moved this up here to be next to the declaration of `g_runtime` and also because this function doesn't capture anything from the definition of `gui`.